habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -1198,42 +1198,42 @@ class HabitatSimFisheyeSemanticSensorConfig(SimulatorFisheyeSensorConfig):
 
 @attr.s(auto_attribs=True, slots=True)
 class HeadRGBSensorConfig(HabitatSimRGBSensorConfig):
-    uuid: str = "robot_head_rgb"
+    uuid: str = "articulated_agent_head_rgb"
```

Comment:
For all of these, I wonder if the articulated_agent prefix is needed. I don't think the robot prefix was needed in the first place.

Reply:
True... @jimmytyyang may have better context.
I agree `articulated_agent` is super verbose. We wanted to make a distinction between this an simply `agent`, but it would be great to find a shorter name...

Reply:
I agree that adding `articulated_agent` or `robot` is unnecessary, especially since `obs_keys` prefaces this with an agent ID. To compare other observation types: `agent_1_articulated_agent_head_depth` versus `agent_0_obj_start_sensor`. To make these consistent, I would remove the `articulated_agent` prefix from all cameras/observations.

Reply:
Addressed!